.travis.yml

```diff
@@ -29,6 +29,7 @@ matrix:
             - *global_deps
             - at-spi2-core
             - build-essential
+            - libgl-dev
```

Comment:
Can you also add this package to our [Github Actions configuration file](https://github.com/gluonhq/substrate/blob/master/.github/workflows/build.yml#L22) ?